src/services/inlayHints.ts

```diff
@@ -267,6 +271,14 @@ namespace ts.InlayHints {
             if (!signature) {
                 return;
             }
+            const shouldWrap = shouldWrapParameterList(node);
+            if (shouldWrap) {
+                result.push({
+                    text: "(",
+                    position: node.parameters[0].getStart(),

```

Comment:
`getStart` takes a `file` argument to be a little bit more efficient.

Reply:
Nice tip, thanks!